lib/CSS/SpriteMaker.pm

```diff
@@ -548,6 +554,7 @@ EONCLICK
             next if $key eq "colors";
             print $fh "<b>" . $key . "</b>: " . ($rh_source_info->{$key} // 'none') . "<br />";
         }
+        if (!$self->{enable_colormap}) {
         print $fh '<h3>Colors</h3>';
             print $fh "<b>total</b>: " . $rh_source_info->{colors}{total} . '<br />';
```

Comment:
I would still print this, as the total number of colors is not determined based on the enable_colormap option